src/components/Home/MainBlob.tsx

```diff
@@ -0,0 +1,35 @@
+import Image, { StaticImageData } from "next/image";
+import React from "react";
+
+interface BlobProps {
+    image: StaticImageData,
+    headline: string,
+    text: string
+}
+
+
+
+export default function MainBlob (props:BlobProps) {
```

Comment:
Just a syntax thing. You can change `(props:BlobProps)` into `({image, headline, text}: BlobProps)` allowing you to use them as variables.